checker/bin-devel/Dockerfile-ubuntu-jdk8

```diff
@@ -20,5 +20,6 @@ RUN apt-get -qqy update && apt-get -qqy install \
   unzip \
   wget \
   default-jdk \
+  binutils \
```

Comment:
The list of packages seems mostly alphabetically sorted. I would add `binutils` after `ant`, in both files.